content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -0,0 +1,593 @@
+    </ul><!--/.breadcrumb.prereqs-->
+  </div><!--/.breadcrumbs-wrap-->
+  <button id="generate-creds-button" class="btn btn-primary">Generate credentials</button>
+  <div id='loader-0' style="display: none;"><img class='throbber' src="assets/img/rippleThrobber.png"> Generating Keys...</div>
+  <div id='address'></div>
+  <div id='secret'></div>
+  <div id='balance'></div>
+  <div id="populate-creds-status"></div>
+</div>
+</div>
+<script type="application/javascript">
+$(document).ready( () => {
+
+  $("#generate-creds-button").click( () => {
+    // Wipe existing results
+    $("#address").html("")
+    $("#secret").html("")
+    $("#balance").html("")
+    $("#populate-creds-status").html("")
+
+    $("#loader-0").show()
+
+    $.ajax({
+      url: "https://faucet.altnet.rippletest.net/accounts",
+      type: 'POST',
+      dataType: 'json',
+      success: function(data) {
+        $("#loader-0").hide()
+        $("#address").hide().html("<strong>Address:</strong> " +
+          '<span id="test-net-faucet-address">' +
+          data.account.address
+          + "</span>").show()
+        $("#secret").hide().html('<strong>Secret:</strong> ' +
+          '<span id="test-net-faucet-secret">' +
+          data.account.secret +
+          "</span>").show()
+        $("#balance").hide().html('<strong>Balance:</strong> ' +
+          Number(data.balance).toLocaleString('en') +
+          ' XRP').show()
+
+        // Automatically populate examples with these credentials...
+        // Set sender address
+        let generated_addr = ""
+        $("code span:contains('"+EXAMPLE_ADDR+"')").each( function() {
+          let eltext = $(this).text()
+          $(this).text( eltext.replace(EXAMPLE_ADDR, data.account.address) )
+        })
+
+        // Set sender secret
+        $("code span:contains('"+EXAMPLE_SECRET+"')").each( function() {
+          let eltext = $(this).text()
+          $(this).text( eltext.replace(EXAMPLE_SECRET, data.account.secret) )
+        })
+
+        $("#populate-creds-status").text("Populated this page's examples with these credentials.")
+
+        // Update all breadcrumbs
+        $(".bc-generate").removeClass("active")
+        $(".bc-generate").addClass("done")
+        $(".bc-connect").removeClass("disabled")
+        $(".bc-connect").addClass("active")
+
+      },
+      error: function() {
+        $("#loader-0").hide();
+        alert("There was an error with the Ripple Test Net, please try again.");
+      }
+    })
+  })
+
+  const EXAMPLE_ADDR = "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe"
+  const EXAMPLE_SECRET = "s████████████████████████████"
+  $("#populate-creds-button").click( () => {
+
+  })
+
+})
+</script>
+
+**Caution:** Ripple operates the XRP Test Net for testing purposes only, and regularly resets the state of the test net along with all balances. As a precaution, Ripple recommends **not** using the same addresses on the Test Net and production.
```

Comment:
Maybe include a note around here explaining that you need to generate a new account to use the demos, because we don't want to risk anybody accidentally including their mainnet secret key, and because you should never give any secret key to a third party.